common/src/main/java/com/hartwig/serve/common/classification/matchers/PositiveMatcher.java

```diff
@@ -0,0 +1,35 @@
+package com.hartwig.serve.common.classification.matchers;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Set;
+
+public class PositiveMatcher implements EventMatcher {
```

Comment:
I don't think we need two classes (`NegativeMatcher` and `PositiveMatcher`). A single `KeyPhraseMatcher` could cover both cases, configured differently for negative and positive.

Reply:
I have create a single matcher which configured the negative and positive events separately

Reply:
Looks good to me.